app/controllers/events/routes.py

```diff
@@ -19,3 +20,33 @@ def showUpcomingEvent():
     upcomingEvents = getUpcomingEventsForUser(g.current_user)
     return render_template('/events/showUpcomingEvents.html',
                             upcomingEvents = upcomingEvents)
+
+
+@events_bp.route('/<eventid>/kiosk', methods=['GET'])
+def loadKiosk(eventid):
+    event = Event.get_by_id(eventid)
+    return render_template("/events/eventKiosk.html",
+                            event = event,
+                            eventid = eventid)
+
+@events_bp.route('/signintoKiosk', methods=['POST'])
+def signinKiosk():
```

Comment:
let's rename these functions (esp. the one in app/logic) to remove the kiosk references. Kiosk mode is how things are displayed, but what's happening here is event sign-in.